cfgov/cdntools/backends.py

```diff
@@ -19,16 +20,23 @@ def __init__(self, params):
         self.client_token = params.get("CLIENT_TOKEN")
         self.client_secret = params.get("CLIENT_SECRET")
         self.access_token = params.get("ACCESS_TOKEN")
+        self.object_id = params.get("OBJECT_ID")
+        self.purge_all_url = params.get("PURGE_ALL_URL")
+        self.fast_purge_url = params.get("FAST_PURGE_URL")
         if not all(
             (
                 self.client_token,
                 self.client_secret,
                 self.access_token,
+                self.object_id,
+                self.purge_all_url,
+                self.fast_purge_url,
             )
         ):
-            raise ValueError(
-                "AKAMAI_CLIENT_TOKEN, AKAMAI_CLIENT_SECRET, "
-                "AKAMAI_ACCESS_TOKEN must be configured."
+            raise ImproperlyConfigured(
+                "Akamai CLIENT_TOKEN, CLIENT_SECRET, ACCESS_TOKEN, "
+                "OBJECT_ID, and PURGE_ALL_URL, and FAST_PURGE_URL "
```

Comment:
```suggestion
                "OBJECT_ID, PURGE_ALL_URL, and FAST_PURGE_URL "
```

The rare double Oxford comma!